_posts/2020-12-18-status-update.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: "2020-12-18: IAC Team Status Update"
+author: sheenaajay
+categories:
+  - community
+  - releases
+  - dsc
+  - windows
+  - cloud-ci
+  - puppet-module-gem
+  - rubocop
+  - infrastructure
+  - team
+  - status
+tags:
+---
```

Comment:
```suggestion
categories:
  - team
  - status
tags:
  - community
  - releases
  - dsc
  - windows
  - cloud-ci
  - puppet-module-gem
  - rubocop
  - infrastructure
---
```

categories are used for constructing the URL, so should only ever be [team, status] for status updates.

Reply:
Thank you